lib/vmpooler/providers/vsphere.rb

```diff
@@ -49,9 +51,7 @@ def name
         end
 
         def folder_configured?(folder_title, base_folder, configured_folders, whitelist)
-          if whitelist
-            return true if whitelist.include?(folder_title)
-          end
+          return true if whitelist&.include?(folder_title)
```

Comment:
what kind of magic is that!

Reply:
the safe navigation operator is probably the coolest thing I saw from this rubocop change

Reply:
That does look rather cool.